test/functional/specs/Personalization/C782718.js

```diff
@@ -95,13 +90,14 @@ test.skip("Test C782718: SPA support with auto-rendering and view notifications"
   }).getPayloadsByType("personalization:decisions");
 
   await t.expect(personalizationPayload.length).eql(3);
-  // here we should verify that we have rendered decisions for page-wide scope and for /products view
+
+  // assert propositions were rendered for both page_load and view
   await t
-    .expect(getDecisionContent("page-wide-scope-SPA_FUNCTIONAL_TEST"))
-    .eql("spa functional test - page wide scope");
+    .expect(getDecisionContent("pageWideScope"))
```

Comment:
I think we are trying to clearly delineate which target items are for functional tests. I'm not sure what the convention is though. @Aaronius, @shammowla do you have an opinion here?

Reply:
@jonsnyder Are these propositions the same propositions that the sandbox displays? If so, we shouldn't be using them in our functional tests. Is that you're talking about @jonsnyder?

Reply:
Oh, I was confusing sandbox propositions with functional test propositions. Nevermind my comment.

Reply:
This was just a test that have failed because of a empty space at the end of the rendered offer content, and it happened only in IE environment. I have removed the empty space from the offer setup itself and the test have resolved.